pkg/spec/v1alpha1/config.go

```diff
@@ -21,10 +21,11 @@ func New() *Config {
 // Config holds the configuration variables for config version v1alpha1
 // ApiVersion and Kind are currently unused, this may change in the future.
 type Config struct {
```

Comment:
this is poorly named

Reply:
Agreed! Before this PR, the term "Config" made sense, but it is definitely incorrect now. I suggest renaming to `Environment" instead.

The `New` constructor may stay the same fwiw, given this package has no other exposed types